### PR TITLE
ZEPPELIN-296 add flink docs

### PR DESCRIPTION
**Pull request by bzz**
[ZEPPELIN-296](https://issues.apache.org/jira/browse/ZEPPELIN-296) initial documentation for Apache Flink interpreter

While we are at it, there are some minor updates for the docs too.

### CODE REVIEW

File: docs/docs/interpreter/flink.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+---------|----------|-----
+local	 | false    | 
+host	 | hostname |
+port	 | 6123     |
+jobmanager.rpc.address | localhost | JobManager host
+jobmanager.rpc.port	   | 6123      | JobManager port
+
+### How to test it's working
+
+I.e by using the [Zeppelin notebook](https://www.zeppelinhub.com/viewer/notebooks/aHR0cHM6Ly9yYXcuZ2l0aHVidXNlcmNvbnRlbnQuY29tL05GTGFicy96ZXBwZWxpbi1ub3RlYm9va3MvbWFzdGVyL25vdGVib29rcy8yQVFFREs1UEMvbm90ZS5qc29u) is from [Till Rohrmann's presentation](http://www.slideshare.net/tillrohrmann/data-analysis-49806564) "Interactive data analysis with Apache Flink" for Apache Flink Meetup.
```
Comment (felixcheung) [inline docs/docs/interpreter/flink.md]: I.e -> i.e. (lower case)

File: docs/docs/interpreter/flink.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+### How to start local Flink clust, to test the interpreter
+Zeppelin comes with pre-configured flink-local interpreter, wich starts Flink in a local mode on your machine, so you do not need to install anything.
+
+### How to configure interpreter to point to Flink cluster
+At the "Interpreters" menu, you have to create a new Flink interpreter and provide next properties:
+
+property | value    | Description
+---------|----------|-----
+local	 | false    | 
+host	 | hostname |
```
Comment (felixcheung) [inline docs/docs/interpreter/flink.md]: is this `"hostname"` or replace with your real hostname?

File: docs/docs/interpreter/flink.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+layout: page
+title: "Flink Interpreter"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+
+## Flink interpreter for Apache Zeppelin
+[Apaceh Flink]() is ...
```
Comment (felixcheung) [inline docs/docs/interpreter/flink.md]: Add this in ()?
https://flink.apache.org/index.html

Comment (chiwanpark) [inline docs/docs/interpreter/flink.md]: [https://flink.apache.org](https://flink.apache.org) would be better.

**Comment by bzz** [comment]
@chiwanpark @felixcheung - thank you for reviews! The feedback has been addressed in 364010c

**Comment by bzz** [comment]
Ready to be merged

**Comment by felixcheung** [comment]
looks good

File: docs/docs/interpreter/flink.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+layout: page
+title: "Flink Interpreter"
+description: ""
+group: manual
+---
+{% include JB/setup %}
+
+
+## Flink interpreter for Apache Zeppelin
+[Apaceh Flink](https://flink.apache.org) is an open source platform for distributed stream and batch data processing.
```
Comment (chiwanpark) [inline docs/docs/interpreter/flink.md]: Typo: Apache

File: docs/docs/interpreter/flink.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+
+### How to start local Flink clust, to test the interpreter
```
Comment (chiwanpark) [inline docs/docs/interpreter/flink.md]: Typo: cluster

File: docs/docs/interpreter/flink.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+Zeppelin comes with pre-configured flink-local interpreter, wich starts Flink in a local mode on your machine, so you do not need to install anything.
```
Comment (chiwanpark) [inline docs/docs/interpreter/flink.md]: Typo: which

**Comment by bzz** [comment]
@chiwanpark thank you! addressed in 8a2f909 
squashed Flink commits

ready to be merged

**Comment by Leemoonsoo** [comment]
Looks good to me.

**Comment by chiwanpark** [comment]
Looks good to merge. :+1:

**Pull request closed without merging**